src/composeappmanager.h

```diff
@@ -5,35 +5,123 @@
 
 #define PACKAGE_MANAGER_COMPOSEAPP "ostree+compose_apps"
 
+// TODO: restructure composeappmanager.* so it contains just the manager implemenation
+// ComposeApp and Docker::* stuff should be moved into separate source files
+
```

Comment:
I agree. I think a docker.cc file for the implementation would be nice. This is only going to grow in complexity.